README.md

```diff
@@ -16,6 +16,10 @@ If you find this interesting, come and discuss on [`#hydrogen:matrix.org`](https
 
 Hydrogen is deployed to [hydrogen.element.io](https://hydrogen.element.io). You can run it locally `yarn install` (only the first time) and `yarn start` in the terminal, and point your browser to `http://localhost:3000`. If you prefer, you can also [use docker](doc/docker.md).
 
```

Comment:
Can we make this more subtle? E.g. just adding this rather than its own section:

```suggestion

Hydrogen uses symbolic links in the codebase, so if you are on Windows, have a look at [making git & symlinks work](https://github.com/git-for-windows/git/wiki/Symbolic-Links) there.
```

Reply:
Yeah that's nice, will commit the suggestion.

Reply:
I think the suggestion got appended rather than getting replaced.

Reply:
Yeah, just remove what you previously wrote and we can merge :+1:

Reply:
Will do it.

Reply:
Added another commit. I think, now it should be fine.